hardhat.config.ts

```diff
@@ -76,7 +84,7 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: {
       bsc: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
-      bsctestnet: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
+      bscTestnet: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
```

Comment:
Is it a requirement from hardhat-etherscan? I would expect these to be sth like
```suggestion
      bscmainnet: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
      bsctestnet: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
```

i.e. the same names as the ones used in the network configuration 🤔